.github/workflows/release.yml

```
@@ -1,28 +1,307 @@
-name: Release
+name: Release Automation
 
 on:
+  push:
+    tags:
+      - 'v*'
   release:
     types: [created]
 
```

This will cause duplicate workflow runs - once on tag push and once on release creation. Choose one trigger:

```suggestion
on:
  push:
    tags:
      - 'v*'
```